apm-agent-plugins/apm-opentelemetry/apm-opentelemetry-plugin/src/main/java/co/elastic/apm/agent/opentelemetry/sdk/OTelSpan.java

```diff
@@ -109,7 +109,7 @@ public Span recordException(Throwable exception, Attributes additionalAttributes
 
     @Override
     public Span updateName(String name) {
-        span.withName(name);
+        span.withName(name, AbstractSpan.PRIO_DEFAULT);
```

Comment:
```suggestion
        span.withName(name, AbstractSpan.PRIO_USER_SUPPLIED);
```

Reply:
oops, must have messed things up before commiting. At least it was caught by my own test :D